langchain/src/llms/tests/bedrock.int.test.ts

```diff
@@ -240,6 +242,33 @@ test("Test Bedrock LLM: stream method", async () => {
     maxTokens: 20,
     region,
     model,
+    async fetchFn(
```

Comment:
Ah I intended this one to test that this works with a live Bedrock instance, without a fetch function mock. Can you try that out?

Reply:
oh sorry, you can revert the mock I added.  I just assumed no connection to bedrock so mocked it.  

To answer your question though, yes I tried this `.stream` against a live bedrock endpoint too.